### DISCUSSION TITLE
Syntax of makefile

### DISCUSSION THREAD

**Discussion by pabloto** [codefori/vscode-ibmi#474] [IBM i] [answered]
Hi, 

I studied your video:

https://www.youtube.com/watch?v=XuiGyWptgDA

I found it very interesting, but I have some problem with the make file, is there some link to explain the correct syntax?

Many thanks and absolutely great work

**Comment by worksofliam** [comment, marked as answer]
These links may be useful.

* https://www.gnu.org/software/make/manual/make.html
* https://worksofbarry.github.io/rpg-git-book/6-buildtool

**Reply by pabloto** [reply]
Many thanks @worksofliam  I'll study it.
